src/actions/accountConfirmation.js

```diff
@@ -3,10 +3,10 @@ import axios from 'axios'
 const accountConfirmation = async (confirmationCode) => {
   try {
     const result = await axios.get(
-      `http://localhost:3001/api/auth/confirm/${confirmationCode}`,
+      `http://clumsy-glasses-clam.cyclic.app/api/auth/confirm/${confirmationCode}`,
+      // `http://localhost:8080/api/auth/confirm/${confirmationCode}`,
```

Comment:
do you need commented out code?

Reply:
I don't need localhost link more. Deleted.